app/assets/stylesheets/general/_titles.scss

```diff
@@ -49,7 +49,21 @@ h6 {
 
 h1, h2, h3, h4, h5, h6 {
   .header_anchor {
-    text-decoration: none;
+    float: left;
+    margin-left: -20px;
+    width: 20px;
+    height: 1em;
+    line-height: 1;
```

Comment:
I feel like the line-height does nothing here

Reply:
Also stolen from GitHub. Indeed it doesn't do anything. I will remove 👍.